### PR TITLE
 Update credentials and scans per mocks, fix redirect warning

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

File: client/src/components/content/content.js [hunk 1]
```diff
@@ -32,8 +32,10 @@ class Content extends React.Component {
 
     return (
       <div className={classes}>
-        <Switch>{renderRoutes}</Switch>
-        {redirectRoot}
+        <Switch>
+          {renderRoutes}
+          {redirectRoot}
+        </Switch>
       </div>
```
Comment (cdcabrera) [inline client/src/components/content/content.js]: Sweet, props to you sir getting rid of the warning!

File: client/src/components/credentials/credentialListItem.js [hunk 2]
```diff
@@ -29,8 +29,10 @@ class CredentialListItem extends React.Component {
     }
 
     let credentialType = 'Username & Password';
-    if (item.ssh_keyfile && item.ssh_keyfile !== '') {
+    if (item.auth_type === 'sshKey') {
       credentialType = 'SSH Key';
+    } else if (item.auth_type === 'becomeUser') {
+      credentialType = 'Become User';
     }
```
Comment (cdcabrera) [inline client/src/components/credentials/credentialListItem.js]: Optional: maybe a switch if we need to add more strings

File: client/src/components/scans/scanListItem.js [hunk 3]
```diff
@@ -2,150 +2,247 @@ import React from 'react';
...
+            >
+              <MenuItem className="unavailable">Pause</MenuItem>
+              <MenuItem className="unavailable">Cancel</MenuItem>
+              <MenuItem divider />
+              <MenuItem className="unavailable">Start Scan</MenuItem>
+            </DropdownKebab>
+          </span>
+        }
+        leftContent={leftContent}
+        heading={'ID: ' + item.id}
```
Comment (cdcabrera) [inline client/src/components/scans/scanListItem.js]: Optional: string template/literal

File: client/src/components/scans/scanListItem.js [hunk 4]
```diff
@@ -2,150 +2,247 @@ import React from 'react';
...
+            >
+              <MenuItem eventKey="1" onClick={onSummaryDownload}>
+                Summary Report
+              </MenuItem>
+              <MenuItem eventKey="2" onClick={onDetailedDownload}>
+                Detailed Report
+              </MenuItem>
+            </DropdownButton>
+            <DropdownKebab
+              id={'dropdownActions_' + item.id}
```
Comment (cdcabrera) [inline client/src/components/scans/scanListItem.js]: Optional: string template/literal

File: client/src/components/scans/scanListItem.js [hunk 5]
```diff
@@ -2,150 +2,247 @@ import React from 'react';
...
+    return (
+      <ListView.Item
+        key={item.id}
+        actions={
+          <span>
+            <DropdownButton
+              bsStyle="primary"
+              title="Download"
+              pullRight
+              id={'downloadButton_' + item.id}
```
Comment (cdcabrera) [inline client/src/components/scans/scanListItem.js]: Optional: string template/literal

File: client/src/components/credentials/credentials.js [hunk 6]
```diff
@@ -47,6 +47,13 @@ class Credentials extends React.Component {
       // Reset selection state though we may want to keep selections over refreshes...
       nextProps.credentials.forEach(credential => {
         credential.selected = false;
+        if (credential.ssh_keyfile && credential.ssh_keyfile !== '') {
+          credential.auth_type = 'sshKey';
+        } else if (credential.become_useer && credential.become_useer !== '') {
```
Comment (cdcabrera) [inline client/src/components/credentials/credentials.js]: is ```.become_useer``` spelled correctly?

**Comment by jeff-phillips-18** [comment]
Updated per review comments. Thanks @cdcabrera

**Pull request merged**